src/Microsoft.TestPlatform.Common/Utilities/FakesUtilities.cs

```diff
@@ -24,7 +24,8 @@ public static class FakesUtilities
 
         private const string CrossPlatformConfiguratorMethodName = "GetCrossPlatformDataCollectorSettings";
 
-        private const string FakesConfiguratorAssembly = "Microsoft.VisualStudio.TestPlatform.Fakes, Version=16.0.0.0, Culture=neutral";
+        private const string FakesConfiguratorAssemblyDev16 = "Microsoft.VisualStudio.TestPlatform.Fakes, Version=16.0.0.0, Culture=neutral";
```

Comment:
Do we still need a 16.0 version?

Reply:
I left both in there because currently we will not be shipping the 17.0 TestPlatform.Fakes in the Microsoft.Testplatform nuget package. We will be able to remove that once we update the TestRunnerHarness project. Do you think there may be any problems in doing so?

Reply:
Oh wouldn't we be able to ship that in the 17.0 package of the TestPlatform? Its a little confusing that we have both versions supported here though in reality we'd only expect one of the versions to be present based on the version the TP package.

Reply:
Ah I did see that we have a 17.0 package already shipped. Updated and removed 16.0